client/index.html

```diff
@@ -9,15 +9,15 @@
 </head>
 <body class="index">
     <div> 
-        <h1 class="indexHeader">
+        <h1 class="google indexHeader">
             <span style="color:#DB4437">N</span><span style="color:#4285F4">o</span><span style="color:#F4B400">t</span>
```

Comment:
i like the not google name, great way to avoid a lawsuit

Reply:
Currently contemplating renaming it to (!Google)